e2e/util_test.go

```diff
@@ -155,35 +175,113 @@ func s3client(t *testing.T, options storage.Options) *s3.S3 {
 	if *flagTestLogLevel == "debug" {
 		awsLogLevel = aws.LogDebug
 	}
+	s3Config := aws.NewConfig()
+
+	id := defaultAccessKeyID
+	key := defaultSecretAccessKey
+	endpoint := options.Endpoint
+	region := endpoints.UsEast1RegionID
+
+	// get environment variables and use external endpoint url.
+	// this can be used to test s3 sources such as gcs, amazon, wasabi etc.
+	if isEndpointFromEnv() {
```

Comment:
I think we should run integration tests if a certain variable is set **and** required parameters are given, such as `S5CMD_I_KNOW_WHAT_IM_DOING=1` :). I'm open to name suggestions.

Reply:
I think that's a great name :)

Reply:
Could you update the PR description as well?